src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -15,7 +15,6 @@ object DplaMapData {
   type ZeroToOne[T] = Option[T]
   type ExactlyOne[T] = T
   type LiteralOrUri = Either[String,URI]
-  type LiteralOrSkos = Either[String,SkosConcept]
```

Comment:
That is true, but also the way you'd make a SkosConcept with only a string would be to assign it to the concept field like `SkosConcept(concept = "a concept")`, not make a union type. Just don't want people to get confused.